examples/drops/backend/.env.template

```diff
@@ -3,3 +3,4 @@ CLIENT_SECRET=
 API_KEY=
 OAUTH_SERVER_DOMAIN=
 POAP_DROPS_BASE_URL=
```

Comment:
`POAP_DROPS_BASE_URL=poap_dropps_base_url`, let's add them like this so we have the values